### ISSUE TITLE
Make slider values more visible

### ISSUE THREAD

**Issue by martalalik** [equinor/design-system#2269]
When hovering over `Slider` to change the values, cursor partly is blocking view of this values.
![IMG_7250](https://user-images.githubusercontent.com/105340174/171808384-29dfce75-2512-4810-865a-940dadbd01c6.jpg)

Although circle element that change the values has space around, that the cursor can be pushed bit higher.
![IMG_7251](https://user-images.githubusercontent.com/105340174/171809424-c0f09cdc-f85a-4302-9d00-1b96d7731348.jpg)

**Comment by mimarz** [comment]
Maybe we can just use the `Tooltip` here? What do you guys think @equinor/eds-ux ?

**Comment by akvam** [comment]
Discussed in backlog meeting. Not high priority, should be discussed when more enhancements are "scoped".

**Comment by oddvernes** [comment]
This will be resolved by #3143
